Source/MQTTnet/Server/MqttSubscriptionInterceptorContext.cs

```diff
@@ -20,8 +22,37 @@ public MqttSubscriptionInterceptorContext(string clientId, TopicFilter topicFilt
         /// </summary>
         public IDictionary<object, object> SessionItems { get; }
 
-        public bool AcceptSubscription { get; set; } = true;
+        private bool acceptSubscription = true;
```

Comment:
Please move the fields to the top of the file and use _ as prefix. This is the default code style of this project.